.github/workflows/markdown-lint-fix.yml

```diff
@@ -43,13 +44,6 @@ jobs:
           repository: mdn/content
           path: mdn/content
```

Comment:
I think this checkout action needs to be moved above the setup-node action to cache appropriately